### PR TITLE
fix(users): profile page is getting other users' models

### PR DESCRIPTION
**Pull request by luabida**
And add a form to enable user to edit their Model on `/models/` page

### CODE REVIEW

File: src/users/forms.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ class UpdateModelForm(forms.Form):
     # model_author = forms.ModelChoiceField(queryset=Author.objects.all())
     model_id = forms.IntegerField()
     model_name = forms.CharField(max_length=100)
-    model_description = forms.CharField(max_length=255)
+    model_description = forms.CharField(max_length=500)
```
Comment (luabida) [inline src/users/forms.py:21]: requested by @eduardocorrearaujo

File: src/registry/api.py [hunk 2]
```diff
@@ -180,6 +180,16 @@ def create_model(request, payload: ModelIn):
         return 403, {"message": "Model repository must be on Github"}
     if not repo_url.path:
         return 403, {"message": "Invalid repository"}
+
+    description = payload.description
+    if len(description) > 500:
+        return 403, {
+            "message": (
+                "Description too big, maximum allowed: 500. "
+                f"Remove {len(description) - 500} characters."
```
Comment (fccoelho) [inline src/registry/api.py:189]: How about a "Please remove..." :-)

File: src/registry/api.py [hunk 3]
```diff
@@ -198,10 +208,7 @@ def create_model(request, payload: ModelIn):
                     f"did you mean '{similar_lang}'?"
                 )
             }
-        else:
-            return 404, {
-                "message": f"Unknown language {payload.implementation_language}"
-            }
+        return 404, {"message": f"Unknown language {payload.implementation_language}"}
```
Comment (fccoelho) [inline src/registry/api.py:211]: Wouldn't be easier to have the implementation language as a select widget in the form?

**Review by fccoelho — CHANGES_REQUESTED** [review]

File: src/registry/api.py (hunk 3 above)
Comment (luabida) [reply src/registry/api.py:211]: Good point, I think it would be way more intuitive also

**Pull request merged**